config/game_options.txt

```diff
@@ -164,7 +164,7 @@ ALLOW_AI_MULTICAM
 
 ## How long the delay is before the Away Mission gate opens. Default is half an hour.
 ## 600 is one minute.
-GATEWAY_DELAY 18000
+GATEWAY_DELAY 60
```

Comment:
I assume that this is an artefact from local testing- just remove it before you un-draft it (or change the above comment associated to the config flag to match and justify it).